novawallet/Modules/NetworkDetails/NetworkDetailsViewModelFactory.swift

```diff
@@ -77,7 +77,8 @@ private extension NetworkDetailsViewModelFactory {
                             ),
                             icon: nil
                         ),
-                        selectable: network.enabled
+                        selectable: network.enabled,
+                        enabled: network.chainId != KnowChainId.polkadot
```

Comment:
How do we ensure that a user can't disable Polkadot network? Can't find how do we manually disabling UISwitch for it

Reply:
there is confusing naming for viewmodel's fields. I've tried to reuse the old one, so `selectable` is actually works like `isOn` and `enabled` is actually for a whole cell ability to be interacted with. That's how we ensure user can't interact with "Enable connection" cell in case of Polkadot network.